repositories/src/test/resources/liquibase/sql/clean-tables.sql

```diff
@@ -0,0 +1,13 @@
+DO
+$func$
+BEGIN
+EXECUTE
+    (SELECT 'TRUNCATE TABLE ' || string_agg(oid::regclass::text, ', ') || ' CASCADE'
+     FROM pg_class
+     WHERE relkind = 'r'
```

Comment:
can you please add a small comment to explain what `relkind = 'r'` means?

Reply:
Done